docs/user-guide/variants.md

```diff
@@ -124,6 +124,36 @@ The out put files of expansion hunter if preferably annotated with [stranger][st
 ### Compounds (top 20)
 Only interesting when the compound inheritance pattern is required, the list can be very long but is by default cropped to the top 20 highest ranked ones as shown in the heading.
 
+## Managed variants
+
+### Managed variants upload file format
+The managed variants csv file is a tab or semicolon (;)-separated text file with a mandatory header that describes the columns and one line for each gene entry. You must use the same delimiter for the whole file. Ideally do not use the delimiter characters in other places in the file. Consult with an admin if you need to use the delimiter characters in other fields for help with escaping them or ensuring a higher priority separator is used on a previous line.
```

Comment:
If it's tab or semicolon-separated perhaps let's not name it csv, otherwise users might think they have to use commas as separators?

Reply:
Sure, this is mostly MS-legacy terminology.